src/widgets/widgets-collection.js

```diff
@@ -16,12 +16,12 @@ module.exports = Backbone.Collection.extend({
     this.bind('change:order', _.debounce(this._onChangeOrder.bind(this), 10), this);
     // If a category model applies the category colors, rest should remove/disable
     // the category colors applied before.
-    this.bind('change:autoStyle', function (m, autoStyle) {
+    this.bind('change:autoStyle', function (model, autoStyle) {
```

Comment:
+1 to getting rid of single letter variable names.

I'd even be more explicit: `model` -> `changedWidgetModel` and `_model` -> `widgetModel` .